packages/odyssey-storybook/src/components/odyssey-mui/CheckboxGroup/CheckboxGroup.stories.tsx

```diff
@@ -68,46 +61,56 @@ const storybookMeta: Meta<
 
 export default storybookMeta;
 
-const SingleTemplate: StoryFn<
-  CheckboxGroupProps & {
-    isDefaultChecked: CheckboxProps["isDefaultChecked"];
-    isIndeterminate: CheckboxProps["isIndeterminate"];
-  }
-> = (args) => (
-  <CheckboxGroup
-    errorMessage={args.errorMessage}
-    hint={args.hint}
-    isDisabled={args.isDisabled}
-    label={args.label}
-  >
-    <Checkbox
-      isDefaultChecked={args.isDefaultChecked}
-      isIndeterminate={args.isIndeterminate}
-      label="Pre-flight systems check complete"
-      name="life-support"
-      value="life-support"
-    />
-  </CheckboxGroup>
-);
-
-export const Single = SingleTemplate.bind({});
-Single.parameters = { controls: { exclude: ["hint", "label"] } };
-Single.args = {};
+const SingleTemplate: StoryObj<CheckboxGroupStoryProps> = {
+  render: (args) => (
+    <CheckboxGroup
+      errorMessage={args.errorMessage}
+      hint={args.hint}
+      isDisabled={args.isDisabled}
+      label={args.label}
+    >
+      <Checkbox
+        isDefaultChecked={args.isDefaultChecked}
+        isIndeterminate={args.isIndeterminate}
+        label="Pre-flight systems check complete"
+        name="life-support"
+        value="life-support"
+      />
+    </CheckboxGroup>
+  ),
+};
 
-export const Checked = SingleTemplate.bind({});
-Checked.parameters = { controls: { exclude: ["hint", "label"] } };
-Checked.args = {
-  isDefaultChecked: true,
+export const Single: StoryObj<CheckboxGroupStoryProps> = {
+  ...SingleTemplate,
+  parameters: {
+    controls: {
+      exclude: ["hint", "label"],
+    },
+  },
+  args: {},
 };
```

Comment:
Oh, this is neat. I figured you could do this with the new `StoryObj` model. Makes sense!